packages/ui/src/components/ContentRenderer.tsx

```diff
@@ -429,7 +431,7 @@ export function InlineContent({
       <Text
         color={color}
         lineHeight="$m"
-        fontSize={viewMode === 'block' ? '$s' : '$m'}
+        fontSize={viewMode === 'block' || viewMode === 'activity' ? '$s' : '$m'}
```

Comment:
Not necessary for this PR, but putting it out there: I wonder whether using styled context/variants would be a cleaner way to handle the conditional tweaks.

Reply:
Strong agree! 

Getting our display logic for chat messages, gallery blocks, and notebook posts to handle `viewMode` more comprehensively would be a cool followup project.